clang/lib/CodeGen/Targets/RISCV.cpp

```diff
@@ -632,6 +633,35 @@ class RISCVTargetCodeGenInfo : public CommonCheriTargetCodeGenInfo {
 
   void setTargetAttributes(const Decl *D, llvm::GlobalValue *GV,
                            CodeGen::CodeGenModule &CGM) const override {
+    if (const auto *VD = dyn_cast_or_null<VarDecl>(D)) {
+
```

Comment:
Remove this empty line

Reply:
[4cc56ac](https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/4cc56ac7f13b779c260a497f6c2844759cb396cc) removes the empty line.